packages/shared/src/utils/stringUtils.ts

```
@@ -136,20 +127,139 @@ export function isUTF8(buf: Buffer): boolean {
+    groupSize?: number;
+  } = {},
+): string {
+  const {
+    chars = randomStringCharsSet.numberAndLetter,
+    groupSeparator = '-',
+    groupSize,
+  } = options;
+
+  // Input validation
+  if (length <= 0) {
+    throw new OneKeyLocalError('Length must be a positive number');
```

ESLint

[error] Expected an error object to be thrown.

(@typescript-eslint/no-throw-literal)